### PR TITLE
Add TargetID in MsgBase and querynode v2 API check

### PR DESCRIPTION
**Pull request by congqixia**
- Add `TargetID` in `MsgBase`
- Add querycoord v2 control API check

See also #19626

Signed-off-by: Congqi Xia <congqi.xia@zilliz.com>

### CODE REVIEW

File: internal/querycoordv2/session/cluster.go [hunk 1]
```diff
@@ -92,6 +93,8 @@ func (c *QueryCluster) LoadSegments(ctx context.Context, nodeID int64, req *quer
 	var status *commonpb.Status
 	var err error
 	err1 := c.send(ctx, nodeID, func(cli *grpcquerynodeclient.Client) {
+		req := proto.Clone(req).(*querypb.LoadSegmentsRequest)
```
Comment (yah01) [inline internal/querycoordv2/session/cluster.go:96]: reason for cloning the request?

File: internal/querycoordv2/session/cluster.go [hunk 2]
```diff
@@ -140,6 +149,10 @@ func (c *QueryCluster) GetDataDistribution(ctx context.Context, nodeID int64, re
 	var resp *querypb.GetDataDistributionResponse
 	var err error
 	err1 := c.send(ctx, nodeID, func(cli *grpcquerynodeclient.Client) {
+		req := proto.Clone(req).(*querypb.GetDataDistributionRequest)
+		req.Base = &commonpb.MsgBase{
```
Comment (yah01) [inline internal/querycoordv2/session/cluster.go:153]: req.Base.TargetID = nodeID?

Comment (congqixia) [reply internal/querycoordv2/session/cluster.go:153]: `req.Base` is nil here

File: internal/querycoordv2/session/cluster.go (hunk 1 above)
Comment (congqixia) [reply internal/querycoordv2/session/cluster.go:96]: Is case of reusing request outside.

File: internal/querycoordv2/session/cluster.go (hunk 2 above)
Comment (yah01) [reply internal/querycoordv2/session/cluster.go:153]: Create `req.Base` outside and only modifying `TargetID` here?

Comment (congqixia) [reply internal/querycoordv2/session/cluster.go:153]: Fixed

**Pull request merged**